src/cljs/main/broadfcui/page/method_repo/method_repo_table.cljs

```diff
@@ -40,17 +59,19 @@
               (not methods) (spinner "Loading...")
               :else
               [Table {:data-test-id "methods-table"
-                      :persistence-key "method-repo-table2" :v 1
+                      :persistence-key "method-repo-table2" :v 2
                       :data methods
                       :style {:content {:paddingLeft "1rem" :paddingRight "1rem"}}
                       :tabs {:style {:margin "-0.6rem -1rem 0.3rem" :padding "0 1rem"
                                      :backgroundColor (:background-light style/colors)}
-                             :items [{:label "Public Methods"
+                             :items [{:label "My Methods"
+                                      :predicate :mine?}
+                                     {:label "Public Methods"
                                       :predicate :public}
-                                     {:label "My Methods"
-                                      :predicate (fn [method]
-                                                   (or (not (:public method))
-                                                       (contains? (set (:managers method)) (utils/get-user-email))))}]}
```

Comment:
is the ordering switch (Public | My vs. My | Public) intentional? I'm guessing this was in a comp.